report/Outline/testPiler.tex

```diff
@@ -11,10 +12,15 @@ \section{Test Generator Plugin}
 What about implementation details that are not part of the knowledge domain of the Mobile Customer Application.
 
 Therefore we envision a compromise from our initial idea. 
-We will still build a plugin, but instead of building a Flutter only plugin, we will build a more general package that can be used either in a Flutter project or in a stand-alone project. 
-This package would then have the ability to connect to the backend and generate the test suite - the output would then be what we will call the test artifacts.
+We will still build a plugin, but instead of building a Flutter only plugin, we will build a more general package that can be used either in a Flutter project or in a stand-alone project.
+Such a general package would allow the development team to have full control over how their tooling pipeline is constructed.
+For some development teams it would be fine to have our solution tight coupled directly into the implementation code base, for other teams though it would be seen as a risc to rely so havily on thirdparty tooling, and they can instead create a simple program that uses our package and thereby decouple their implementation code base from our package. 
```

Comment:
```suggestion
For some development teams it would be fine to have our solution tight coupled directly into the implementation code base, for other teams, it would be seen as a risk to rely so heavily on third party tooling, and they can instead create a simple program that uses our package and thereby decouple their implementation code base from our package. 
```